cmd/cstor-volume-grpc/app/command/commands.go

```diff
@@ -17,137 +17,20 @@ limitations under the License.
 package command
 
 import (
-	"context"
-	"encoding/json"
-	"errors"
 	"flag"
 	"fmt"
-	"strings"
 
 	"github.com/golang/glog"
-	"github.com/openebs/maya/cmd/cstor-volume-grpc/api"
-	"github.com/openebs/maya/pkg/client/generated/cstor-volume-grpc/v1alpha1"
+	grpc_util "github.com/openebs/maya/pkg/grpc"
```

Comment:
can `grpc` be the alias

Reply:
in addition can the code be written under `pkg/grpc/v1alpha`

Reply:
will do.